rpc/backend/account_info.go

```diff
@@ -172,14 +173,27 @@ func (b *Backend) GetBalance(address common.Address, blockNrOrHash rpctypes.Bloc
 
 // GetTransactionCount returns the number of transactions at the given address up to the given block number.
 func (b *Backend) GetTransactionCount(address common.Address, blockNum rpctypes.BlockNumber) (*hexutil.Uint64, error) {
+	n := hexutil.Uint64(0)
```

Comment:
won't the `getAccountNonce` grpc query return error when query in future?

Reply:
it was return `0x0` if account contains no transaction